app/experimenter/nimbus-ui/README.md

```diff
@@ -236,15 +236,29 @@ While an experiment is in `draft` status all fields are technically allowed to b
 
-## Access Visualization Data Locally
+## Results Page and Visualization Data
 
 The Results page renders experiment analysis data and can only be accessed when an experiment is complete, the analysis feature flag is on, and the analysis data returned from the visualization API endpoint is available.
 
+### Locally Access Visualization Data for Experiments in Prod
+
+At the time of writing, locally created complete experiments don't return data from the visualization endpoint. However, it is possible to view a locally created experiment that mirrors an experiment in production to test the Results page locally with production visualization data.
+
 To see the Results page and data locally:
 
 1. Ensure your [Google Credentials](https://github.com/mozilla/experimenter#google-credentials) are configured
 2. Set `FEATURE_ANALYSIS=true` in your local `.env` file
-3. Create a Nimbus Experiment at `localhost/admin` with experiment slug "testing-nimbus" status "complete," a branch/branch slug named "control", and another branch/branch slug named "treatment". Alternately, change an existing, complete Nimbus experiment with branches in place to reflect the experiment slug `testing-nimbus`.
+3. Find a completed experiment in production that has analysis data ready that you'd like to test your changes against
+4. At `localhost/admin`, add a new Nimbus Experiment with the same experiment slug as the production experiment you'd like to test against. Change the status to "complete," and add every branch (name and slug) to match the production experiment. Also, make sure "reference branch" is set to the reference branch for that production experiment.
+5. Optionally change `fetch_jetstream_data` in `app/experimenter/settings.py` to `30` so it fetches and stores the data in the local database
+
+You should now be able to navigate to `localhost/nimbus/your-experiment-slug/results` to test your changes locally.
+
+### Jetstream Metadata and Caching
+
+With the data returned from the visualization endpoint, a `metadata` object is also returned. At the time of writing, it contains a `description` per metric used for tooltips on the Results page as well as an `external_config` object containing some experiment properties than can be overridden for analysis purposes only. Jetstream configs for experiments are located in the [`jetstream-config`](https://github.com/mozilla/jetstream-config/) repository and the metadata schema lives in the [Jetstream repo](https://github.com/mozilla/jetstream/).
+
+Visualization data is cached and refetches every 8 hours for experiments less than 3 days old. After that point, experiment data is not refetched. If a manual refetch needs to occur, in the production `/admin`, go to Nimbus Experiments > Action > "Force jetstream data fetch".
```

Comment:
it refetches for experiments that are live or whose end date is less than 3 days ago.

Reply:
Oh oops, that is totally what I meant when I wrote this. 😆